arch/x86/core/ia32/intstub.S

```diff
@@ -371,6 +372,7 @@ SECTION_FUNC(PINNED_TEXT, z_SpuriousIntHandler)
 
 	/* re-enable interrupts */
 	sti
+	nop
```

Comment:
This is safe, the call immediately below is hitting a link-time symbol that won't change at runtime, so it doesn't matter whether the CALL instruction (whose externally visible side effect is just to push a return value onto the stack) happens before or after the pending interrupt.